.github/actions/test-e2e/action.yml

```diff
@@ -0,0 +1,13 @@
+name: E2E tests
```

Comment:
Probably rename this to "Integration tests". Technically this isn't "e2e" with a fully built-out back-end.